src/app/pages/home/components/learning-page-item/learning-page-item.component.scss

```diff
@@ -50,14 +50,21 @@ learning-item {
     padding-right: 5px;
     padding-bottom: 5px;
     padding-left: 5px;
-    font-size: 12px;
+    padding-top: 5px;
+    text-align: start;
+    font-size: 14px;
+
+    a {
```

Comment:
why do we have two anchor class?

Reply:
I didn't check the previous style. Just add a style to resolve the related issue.